docs/mine/spacerace.md

```diff
@@ -187,6 +187,26 @@ lotus-miner sectors status --on-chain-info $SECTOR_NUMBER | grep OnTime
 
 - If the deal is marked as successful on the dashboard but the above commands do not return the expected result, simply repeat steps #1-3 above.
 
+#### How do I force a WaitDeals sector to start sealing?
+
+Run `lotus-miner sectors seal <sector id>`.
+
+#### How is the deal rate of the combined miners being calculated?
```

Comment:
Let's remove as a standalone FAQ. I added a commit to line 112 that clarifies this, and we can point miners to that in the future.